data/en/qcas_0018.json

```diff
@@ -0,0 +1,726 @@
+                            "answers_to_calculate": [
+                                "machinery-equipment-proceeds-answer-1",
+                                "machinery-equipment-proceeds-answer-2",
+                                "machinery-equipment-proceeds-answer-3",
+                                "machinery-equipment-proceeds-answer-4",
+                                "machinery-equipment-proceeds-answer-5",
+                                "machinery-equipment-proceeds-answer-6"
+                            ],
+                            "titles": [{
+                                "value": "Total value of proceeds from disposals"
+                            }]
+                        }
+                    }, {
+                        "id": "any-significant-changes-block",
+                        "title": "Significant changes",
+                        "type": "Question",
+                        "questions": [{
+                            "id": "any-significant-changes-question",
+                            "title": "Did any significant changes occur to the total investment for {{ metadata['trad_as_or_ru_name'] }}?",
+                            "definitions": [{
+                                "title": "What constitutes a significant change?",
+                                "content": [{
+                                    "description": "What constitutes a ‘significant change’ is dependent on your own interpretation in relation to {{ metadata['trad_as_or_ru_name'] }}’s figures from the previous reporting period and the same reporting period last year."
+                                }]
+                            }],
+                            "type": "General",
+                            "answers": [{
+                                "id": "any-significant-changes-answer",
+                                "mandatory": true,
+                                "type": "Radio",
+                                "label": "",
+                                "q_code": "146a",
+                                "options": [{
+                                        "label": "Yes",
+                                        "value": "Yes"
+                                    },
+                                    {
+                                        "label": "No",
+                                        "value": "No"
+                                    }
+                                ]
+                            }]
+                        }],
+                        "routing_rules": [{
+                                "goto": {
+                                    "block": "significant-changes-reason-block-1",
+                                    "when": [{
+                                        "id": "any-significant-changes-answer",
+                                        "condition": "not equals",
+                                        "value": "No"
+                                    }]
```

Comment:
`not equals no` you mean `equals yes`. Although it's not wrong we should probably avoid the double negative.